core/src/cg/group4/Launcher.java

```diff
@@ -69,10 +74,16 @@ public final void create() {
         cTimeKeeper = TimerStore.getInstance().getTimeKeeper();
 
         cStandUp = StandUp.getInstance();
+        
+        //Needs to be moved
+        Collection c = new Collection();
+        c.add(new FishA(780));
+        c.add(new FishB(580));
+        c.add(new FishC(380));
 
         ScreenStore cScreenStore = ScreenStore.getInstance();
         setScreen(cScreenStore.getWorldRenderer());
-        cScreenStore.init();
+        cScreenStore.init(c);
```

Comment:
A screenstore should not need a collection.. At the creation of a collectible screen it should retrieve the players collection. Or should subscribe to it or something..